config/locales/affordability.en.yml

```diff
@@ -42,15 +42,17 @@ en:
         mortgage_calculator/person:
           raw_annual_income: "Your annual income"
           raw_monthly_net_income: "Your monthly take-home pay"
-          annual_income: "Your annual income"
-          extra_income: "Any other annual income"
-          monthly_net_income: "Your monthly take-home pay"
+          annual_income: "What is your annual income or salary before tax?"
+          extra_income: "Do you have any other income? (Optional)"
+          monthly_net_income: "What is your monthly take-home pay?"
+          monthly_net_sub: "(this is your monthly income after tax, minus pension, NI and other deductions)"
         mortgage_calculator/person_other:
           raw_annual_income: "Second applicant's annual income"
           raw_monthly_net_income: "Second applicant's monthly take-home pay"
-          annual_income: "Second applicant's annual income"
-          extra_income: "Second applicant's other annual income"
-          monthly_net_income: "Second applicant's monthly take-home pay"
+          annual_income: "Second applicant's annual income or salary before tax?"
+          extra_income: "Does the second applicant have any other income? (Optional)"
+          monthly_net_income: "Second applicant's monthly take-home pay?"
+          monthly_net_sub: "(dyma yw eich incwm misol ar ôl treth, meinws pensiwn, Yswiriant Gwladol, a didyniadau eraill)"
```

Comment:
This has a similar problem to the English version, but yours and theirs is transposed. Either way there needs to be a separate entry for each.

Reply:
@davidtrussler The copy has been changed to questions and the "yours" issue has been resolved.